onnxruntime/core/platform/windows/env.cc

```diff
@@ -68,31 +69,53 @@ class WindowsThread : public EnvThread {
     int index;
     unsigned (*start_address)(int id, Eigen::ThreadPoolInterface* param);
     Eigen::ThreadPoolInterface* param;
-    const ThreadOptions& thread_options;
+    size_t affinity_mask;
     Param(const ORTCHAR_T* name_prefix1,
           int index1,
           unsigned (*start_address1)(int id, Eigen::ThreadPoolInterface* param),
-          Eigen::ThreadPoolInterface* param1,
-          const ThreadOptions& thread_options1) : name_prefix(name_prefix1), index(index1), start_address(start_address1), param(param1), thread_options(thread_options1) {}
+          Eigen::ThreadPoolInterface* param1) 
+      : name_prefix(name_prefix1),
+      index(index1),
+      start_address(start_address1),
+      param(param1),
+      affinity_mask(std::numeric_limits<size_t>::max()) {}
```

Comment:
>(std::numeric_limits<size_t>::max( [](http://example.com/codeflow?start=19&length=34)

One more thing merit consideration here - so we take size::max() to indicate the disabling of affinity setting, but a thread would be legitimately to have that max() as a valid mask to run on all cores. Conversely, might 0 be a better option?